docs/integrations/labext.md

```diff
@@ -0,0 +1,68 @@
+# Jupyter Lab Extension
+
+Now you can commit your Jupyter Notebook to [Jovian](https://jovian.ml?utm_source=docs) with just **One Click**.
+Make sure you’ve completed the [Installation](../user-guide/01-install.md) before reading further.
```

Comment:
This does not install lab extension. Can you discuss with mayank and update lab installation here

Reply:
Yes, I will talk with Mayank and update this lab installation later. I will keep this empty for now.